### PR TITLE
Sound integration OperatingSystem UI

### PR DESCRIPTION
**Pull request by xrgranados**
Sound integration of the new OperatingSystem UI

https://phabricator.endlessm.com/T25315

### CODE REVIEW

**Comment by cosimoc** [comment]
Needs to be rebased

File: data/metadata.json [hunk 1]
```diff
@@ -714,5 +714,70 @@
         "sound-file": "lightspeed/bg/zarathustra_whale.webm",
         "loop": true,
         "type": "bg"
+    },
+    "operatingSystem/daemons": {
```
Comment (cfoch) [inline data/metadata.json]: We are using the "system" folder for OperatingSystem sounds.

Comment (cfoch) [reply data/metadata.json]: So put the sounds there.

**Comment by cfoch** [comment]
You could have squashed your commits into one. Luckily Github has an option "squash and merge" :)

**Pull request merged**